event-sourcing/subscription-manager/src/test/java/uk/gov/justice/services/event/sourcing/subscription/SubscriptionDescriptorRegistryTest.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.justice.services.event.sourcing.subscription;
+
+import static com.google.common.collect.ImmutableMap.of;
+import static java.util.Collections.EMPTY_MAP;
+import static java.util.Collections.emptyEnumeration;
+import static java.util.Optional.empty;
+import static org.junit.Assert.*;
```

Comment:
Should be specific about what is imported

Reply:
Updated